pkg/porter/archive.go

```diff
@@ -170,26 +165,131 @@ func (ex *exporter) export() error {
 	return err
 }
 
-// chtimes updates all paths under the provided archive path with a constant
-// atime and mtime (Unix time 0), such that the shasum of the resulting archive
-// will not change between repeated archival executions using the same bundle.
-// See: https://unix.stackexchange.com/questions/346789/compressing-two-identical-folders-give-different-result
-func (ex *exporter) chtimes(path string) error {
-	err := filepath.Walk(path,
-		func(subpath string, info os.FileInfo, err error) error {
+func (ex *exporter) createTarHeader(ctx context.Context, path string, file string, fileInfo os.FileInfo) (*tar.Header, error) {
+	log := tracing.LoggerFromContext(ctx)
```

Comment:
Just a suggestion/fyi but a couple ways to make this easier to trace/troubleshoot would be to either create a span per call to createTarHeader, and then write the relative path and the header name as attributes, or to write a log line per call and then again use attributes on the log.

Right now the log.Debugf calls will print to the console if they are running with verbosity=debug, and it will just be kind of rando info being printed. By using a more structure with a span and attributes, it will be easier for someone to troubleshoot later, without dumping contextless debug lines to the console.

Reply:
Hmm.. I don't get any debug statement printed when I archive a bundle. They are however added to the OTEL trace (even with verbosity=info) as log statements for the active span:
![image](https://user-images.githubusercontent.com/15995839/211270508-7a72785d-26b3-4d8d-b9d7-11e349608c7a.png)

Reply:
There is an open issue to fix setting verbosity via a flag, #2505, sorry about that!